css-canvas-medical/product.css

```diff
@@ -0,0 +1,327 @@
+.column-header-main {
+  border-bottom: 2px solid rgb(240, 240, 240);
+}
+
+.row-header {
+  display: flex;
+  width: 87%;
+  margin: auto;
+  height: 70px;
+}
+
+.column-header {
+  width: 100%;
+  margin: auto;
+  display: flex;
+}
+
+.column-nav {
+  width: 120%;
+  position: relative;
+  color: #b3b3b3;
+  font-size: 0.8rem;
+  font-weight: bolder;
+  text-align: center;
+}
+
+.column-nav-blog {
+  width: 80%;
+}
+
+.column-logo {
```

Comment:
Your column classes still seem to be named after the element they are being applied to rather than their measurement. Please review `css-layout-classes` if you are feeling unclear about how to properly construct layout classes that describe their purpose. Layout classes are utility classes, which means they should have limited styling applied to them, and their name should describe their purpose, such as `column-half`, `column-third`, etc.